### PR TITLE
Finish RPM packaging

### PR DESCRIPTION
**Pull request by bbockelm**
This contains the remaining TODO items for a proper RPM package.

### CODE REVIEW

File: configs/50-scitokens-credmon.conf [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# LOCAL_CREDMON_KEY_ID = local
+
+# This instructs the CreDD to utilize "OAuth2 mode" of managing tokens
+# (tokens are all generated on the submit host)
+CREDD_OAUTH_MODE = True
+
+# Common directories and definition of the various credmon-related daemons
+SEC_CREDENTIAL_DIRECTORY = /var/lib/condor/credentials
+SEC_CREDENTIAL_MONITOR = /usr/bin/scitokens_credmon
+SEC_CREDENTIAL_MONITOR_LOG = /var/log/condor/CredMon
```
Comment (jasoncpatton) [inline configs/50-scitokens-credmon.conf]:
```suggestion
SEC_CREDENTIAL_MONITOR_LOG = $(LOG)/CredMonLog
```
Matches convention.

**Review by jasoncpatton — CHANGES_REQUESTED** [review]
I'm not sure if we actually need to be setting up logging in the wsgi script. Flask should already have a handler set up that `OAuthCredmonWebserver` should use automatically. But I should test this...

Also, the `OAuthCredmonWebserver` may not always have write access to `SEC_CREDENTIAL_DIRECTORY`, so we may need to eventually move where the `app.secret_key` gets written.

File: rpm/python-scitokens-credmon.spec [hunk 2]
```diff
@@ -41,18 +43,23 @@ rm -rf %{pypi_name}.egg-info
...
 %files -n python2-%{pypi_name} 
 %doc 
-%{_bindir}/condor_credmon
-%{_bindir}/condor_credmon-2
-%{_bindir}/condor_credmon-%{python2_version}
+%{_bindir}/scitokens_credmon
+%{_bindir}/scitokens_credential_producer
 %{python2_sitelib}/credmon
 %{python2_sitelib}/scitokens_credmon-%{version}-py?.?.egg-info
+%attr(0750, root, condor) /var/lib/condor/credential
```
Comment (jasoncpatton) [inline rpm/python-scitokens-credmon.spec]:
```suggestion
%attr(2770, root, condor) /var/lib/condor/credential
```
Enable the `OAuthCredmonWebserver` to write to the credential dir (and enable SetGID).

File: configs/scitokens_credmon.conf [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+<VirtualHost _default_:8443>
+
+  SSLEngine on
+
+  WSGIScriptAlias / /var/www/wsgi-scripts/scitokens-credmon.wsgi
+
+  # A modest number of processes and threads; production applications may want to increase this.
+  WSGIDaemonProcess     CondorCredmon user=condor group=condor processes=2 threads=25
+  WSGIProcessGroup      CondorCredmon
+  WSGIScriptAlias       / /var/www/cgi-bin/wsgi/scitokens-credmon/scitokens-credmon.wsgi
```
Comment (jasoncpatton) [inline configs/scitokens_credmon.conf]: Ignorant question: Is `/var/www/cgi-bin/wsgi` a typical location for WSGI scripts?

Comment (bbockelm) [reply configs/scitokens_credmon.conf]: I’m not sure if there’s a great convention - I simply copied from a separate project.

Comment (bbockelm) [reply configs/scitokens_credmon.conf]: I did end up simplifying the directory path in the end, but still failed to find any consistency out there in RPM-land.

**Comment by bbockelm** [comment]
@jasoncpatton - I think this is ready to go in.  Please re-review.

**Review by jasoncpatton — APPROVED** [review]

**Pull request merged**